components/App/index.js

```diff
@@ -18,14 +19,18 @@ export default class App extends Component {
     }
 
     createBoard() {
-        this.setState({boards: [...this.state.boards, {name: this.state.boardInput}]});
+        this.setState(s => {return {boards: [...s.boards, {name: s.boardInput}]}});
```

Comment:
it's more readable to do:
`this.setState(s => ({boards: [...s.boards, {name: s.boardInput}]}));`